packages/edit-site/src/components/list/table.js

```diff
@@ -18,19 +18,32 @@ import Actions from './actions';
 import AddedBy from './added-by';
 
 export default function Table( { templateType } ) {
-	const { records: templates, isResolving: isLoading } = useEntityRecords(
+	const { records: allTemplates } = useEntityRecords(
 		'postType',
 		templateType,
 		{
 			per_page: -1,
 		}
 	);
+
+	const templates = useSelect(
+		( select ) =>
+			allTemplates?.filter( ( template ) =>
+				select( coreStore ).isDeletingEntityRecord(
```

Comment:
@annziel  I missed the missing negation here: 

```suggestion
				! select( coreStore ).isDeletingEntityRecord(
```

Reply:
Ups, thank you!